utility

```diff
@@ -0,0 +1 @@
+modem
```

Comment:
This is an odd artifact of the arduino IDE. It is a symlink because the IDE will only compile .cpp files that are either in the root of a library or in a folder called "utility". 

In the long run I am not sure what we want to do. The modem is logically completely separate from the Firebase library so I don't really want to dump them both in a flat folder. 

What do you think of separating them into different arduino libraries? Maybe even different github repos?